docs/mine-hnt/validators/validator-deployment-guide.mdx

```diff
@@ -421,13 +445,13 @@ After the validator has been up and running for 15 minutes, proceed with some of
 The [Validator API](/api/blockchain/validators) provides several useful calls to help monitor your Validator and the state of the Testnet.
 
 ```
-https://testnet-api.helium.wtf/v1/validators/
+https://api.helium.wtf/v1/validators/
```

Comment:
The correct URL is probably `https://api.helium.io/v1/validators/`
But I don't know when the validator endpoint will be live there, maybe @madninja  knows?

Same comment for the URL below.